src/cli.ts

```diff
@@ -92,24 +94,39 @@ const argv = yargs(process.argv.slice(2)).options({
 }).argv;
 
 (async () => {
+  async function closeProgram() {
+    await modelServer.stop();
+
+    process.exit(processStatus);
+  }
+
   const modelServer = new FileServer(path.dirname(argv.input));
+  let options: CaptureScreenShotOptions;
+  let processStatus = 0;
 
   await modelServer.start();
 
-  const options = prepareAppOptions({
-    modelPort: modelServer.port,
-    argv,
-  });
+  try {
+    options = await prepareAppOptions({
+      modelPort: modelServer.port,
+      argv,
+    });
+  } catch (error) {
+    logError(error);
+    processStatus = 1;
+  }
+
+  if (processStatus !== 0) {
+    closeProgram();
+    return;
+  }
 
-  let processStatus = 0;
   try {
-    await captureScreenshot({...options, devicePixelRatio: 1.0});
+    await captureScreenshot(options);
   } catch (err) {
-    console.log(`❌ Unhandled: ${err}`);
+    logUnhandledError(err);
     processStatus = 1;
   }
 
-  await modelServer.stop();
-
-  process.exit(processStatus);
+  await closeProgram();
```

Comment:
why `await`? 
in L120 it's called it without, I guess `process.exit` won't be returning anything 😅